sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -202,6 +232,39 @@ private Message createDispositionMessage(UUID[] lockTokens, DispositionStatus di
         return message;
     }
 
+    private Flux<Instant> renewMessageLock(UUID[] renewLockList) {
+
+        return  isAuthorized(PEEK_OPERATION_VALUE).thenMany(createRequestResponse.flatMap(channel -> {
+            UUID[] lockTokens = Arrays.stream(renewLockList)
+                //.map(ServiceBusReceivedMessage::getLockToken)
```

Comment:
Is there a reason you're creating another array with the same content?

Reply:
removing it.